test-suite/test-functionality.sh

```diff
@@ -260,5 +260,5 @@ main() {
     run_tests $tests
 }
 
-main
+main "$@"
```

Comment:
FWIW, I forgot to add these parameters when I moved the code above into main(), a function created while polishing the original PR [1160](https://github.com/squid-cache/squid/pull/1160/files#r986398359) code.